kaznet/apps/ona/api.py

```diff
@@ -295,6 +322,32 @@ def fetch_form_data(  # pylint: disable=too-many-arguments
     return request(url=url, method='GET', args=query_params)
 
 
+def sync_submission_review(instance_id, ona_review_status, comment):
```

Comment:
Can we please add [type hints](https://docs.python.org/3/library/typing.html) here?